### ISSUE TITLE
[Feature Request] Visual examples of phonometrica features

### ISSUE THREAD

**Issue by atakanokan** [phonometrica/phonometrica#10]
Could you add some visual examples of how the tool is used/going to be used?

**Comment by jeychenne** [comment]
Yes, this is definitely on our roadmap. We'll work on improving the documentation (including screencasts) once we have implemented formant queries, which is the feature we are working on at the moment.
